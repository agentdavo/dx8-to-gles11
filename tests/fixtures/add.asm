add r0, v0, v1
