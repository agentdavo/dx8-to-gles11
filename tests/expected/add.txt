TEX_ENV_COMBINE func=260
